examples/scikit-learn/species_distribution_modeling/README.md

```diff
@@ -0,0 +1,43 @@
+# Species distribution modeling
+
+We translate the Species distribution modeling from scikit-learn into Hamilton to showcase `pipe` and`post_pipe`.
+
+# Highlights
+ Example of a simple ETL pipeline broken into modules with external couplings.
+
+1) To see how to couple external modules / source code and integrate it into a Hamilton DAG check out `grids.py` or `preprocessing.py`, where we use `@pipe` to wrap and inject external functions as Hamilton nodes.
```

Comment:
Example is really good -- I'd highlight a litlte more use of pipe in the readme (E.G. why we're using it + a small example).

ALso there are some single-step pipe calls -- might be nice to see if they should be double ones? 

Furthermore, the pipe calls end up showing up in the graph... I think starting them with underscores should fix that?

Reply:
Re@ renaming: just to double check it should be `pipe_outputs` and not `pipe_output`? One is singular the other plural then

Reply:
Re hiding node: I added the underscore but the nodes didn't disappear for me which is strange, since they did before.. Need to see if there's a problem for me only or something else going haywire.